api/src/test/java/net/kyori/adventure/util/HSVLikeTest.java

```diff
@@ -33,6 +34,14 @@
 import java.math.RoundingMode;
 
 class HSVLikeTest {
+  @Test
+  void roundTripRgbHsvTest() {
+    NamedTextColor.NAMES.values().forEach(namedTextColor -> {
+      final TextColor roundTripped = TextColor.color(namedTextColor.asHSV());
+      Assertions.assertEquals(namedTextColor, roundTripped);
```

Comment:
Should be `static` import if not already one:
```suggestion
      assertEquals(namedTextColor, roundTripped);
```